app/config/Constants.scala

```diff
@@ -16,7 +16,10 @@
 
 package config
 
+import models.DepartureId
+
 object Constants {
-  val BoxName         = s"customs/transits##1.0##notificationUrl"
-  val XClientIdHeader = "X-Client-Id"
+  def requestId(departureId: DepartureId): String = s"/customs/transits/movements/departures/${departureId.index}"
```

Comment:
Amaal asked if we could move this into the `DepartureMessageNotification` companion object as a `private def` which seems like a perfectly reasonable request to me, let's quickly do that tomorrow before merging